include/version.mk

```diff
@@ -56,6 +56,10 @@ VERSION_PRODUCT:=$(if $(VERSION_PRODUCT),$(VERSION_PRODUCT),Generic)
 VERSION_HWREV:=$(call qstrip,$(CONFIG_VERSION_HWREV))
 VERSION_HWREV:=$(if $(VERSION_HWREV),$(VERSION_HWREV),v0)
 
+# borrowed from include/target.mk and include/image.mk
```

Comment:
I don't thinks this line is needed since `git blame` shows the missing info.

Reply:
Done